vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/WebElementsSteps.java

```diff
@@ -197,6 +203,59 @@ public void isFrameWithCertainAttributeFound(State state, String attributeType,
         baseValidations.assertElementState("The found frame is " + state, state, element);
     }
 
+    /**
+     * Gets a list of elements by <b>locator</b> and checks that it is
+     * sorted by text with using <b>sortingOrder</b>.
+     * The list should have more than 1 element. If some elements don't
+     * contain text it will be broken.
+     * @param locator locator to locate elements
+     * @param sortingOrder sorting order<br>
+     * <i>Available rule:</i>
+     * <ul>
+     * <li><b>ASCENDING</b> - will check that the list is sorted alphabetically,
+     * <li><b>DESCENDING</b> - will check that the list is sorted reverse alphabetically
+     * </ul>
+     */
+    @Then("elements located `$locator` are sorted by text in $sortingOrder order")
+    public void areElementSorted(Locator locator, SortingOrder sortingOrder)
+    {
+        List<WebElement> elements = searchActions.findElements(locator);
+        if (softAssert.assertTrue(String.format("The elements by locator: %s", locator), elements.size() >= 2))
```

Comment:
use `baseValidations` to assert elements number (but please bear in mind, do not use deprecated API)